Update github codeowners guide (#2211)
port-labs/port-docs: docs/guides/all/connect-github-codeowners-with-service-team-and-user.md

                            | gsub("@" ; "")
                          )
                  ),
                teams: (
                    $tokens[1:]
                    | map(select(test("^@[^ ]+/[^ ]+$"))
                          | split("/")
                          | .[-1]
              ))
            }
            )
          )
      entity:
        mappings:
          identifier: .repo.full_name + "_" +.item.identifier + "_codeowners"
          title: .item.scope + " codeowners"
          blueprint: '"githubCodeowners"'
          properties:
            scope: .item.scope
            location: .file.path
          relations:
            repository: .repo.full_name
            teams: 
              combinator: '''and'''
              rules:
                - property: '"$title"'
                  value: .item.teams
                  operator: '"in"'
            users: .item.users

```

</details>

Using this method applies the configuration to all repositories that the GitHub app has permissions to.
When configuring the integration **using Port**, the YAML configuration is global, allowing you to specify mappings for multiple Port blueprints.

> Next, add content to your `CODEOWNERS` file, then click on `Resync` and wait for the entities to be ingested to your Port environment.

Reviewer: this should be under its own header.
Please also put the blueprint & mapping sections under a "Set up data model" header like we have in other guides.

<details>
<summary><b>CODEOWNERS file example (Click to expand)</b></summary>

``` showLineNumbers
# Global owners for the entire repository
*        @org-name/global-team

# Documentation files
/docs/   @org-name/docs-team @docsTeamLead

# Frontend code ownership
/frontend/   @org-name/frontend-team @frontendTeamLead

# Backend code ownership
/backend/   @org-name/backend-team @backendTeamLead

# Configuration files
*.yml    @org-name/config-team

# Scripts and utilities
/scripts/ @org-name/ops-team @scriptsHandler
```

</details>

> Now you can see your GitHub CODEOWNERS entities in your catalog.

Reviewer: ```suggestion
In your [software catalog](https://app.getport.io/organization/catalog), the Codeowners page will now display your Codeowners entities:
```
